config/locales/de.yml

```diff
@@ -0,0 +1,64 @@
+# Files in the config/locales directory are used for internationalization
+# and are automatically loaded by Rails. If you want to use locales other
+# than English, add the necessary files in this directory.
+#
+# To use the locales, use `I18n.t`:
+#
+#     I18n.t 'hello'
+#
+# In views, this is aliased to just `t`:
+#
+#     <%= t('hello') %>
+#
+# To use a different locale, set it with `I18n.locale`:
+#
+#     I18n.locale = :es
+#
+# This would use the information in config/locales/es.yml.
+#
+# To learn more, please read the Rails Internationalization guide
+# available at http://guides.rubyonrails.org/i18n.html.
+
+de:
+  #controlle part
+  word_create_success: "Das Wort wurde erfolgreich erstellt"
+  word_update_success: "Das Wort wurde erfolgreich geändert"
+  delete_confirmation: "Wirklich löschen?"
+
+  #word view part
+  word_name: "Name"
+  word_description: "Beschreibung"
+  word_syntactical_category: "Syntax-Kategorie"
+  word_semantical_category: "Semantische Kategorie/en"
+  words_overview: 'Übersicht Wörter'
+
+
+  #general stuff
+  save: "speichern"
+  edit: "bearbeiten"
+  delete: "löschen"
+  back: "zurück"
+  show: 'anzeigen'
+  backward: 'zurück'
+  forward: 'vorwärts'
+  confirmation_question: 'Sind Sie sich sicher?'
+
+
+  #navbar section
+  about: "Über uns"
+  category_library: "Categorien"
+  new_category: "Neue Categorie"
+  card_libary: "Karteikarten"
+  new_card: "Neue Karteikarte"
+  login: "Login"
+
+  #footer
+  company: 'eonum AG'
+  street: 'Effingerstrasse 55'
```

Comment:
Texts that cannot be localized do not need to be internationalized: "Effingerstrasse 55" is the same on all languages. '3008' as well. Though 'Bern' can be internationalized as it has different names in other languages ('Berne', 'Berna'..)